src/main/java/net/sf/mzmine/util/R/RSessionWrapperException.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2006-2014 The MZmine 2 Development Team
+ * 
+ * This file is part of MZmine 2.
+ * 
+ * MZmine 2 is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ * 
+ * MZmine 2 is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with
+ * MZmine 2; if not, write to the Free Software Foundation, Inc., 51 Franklin St,
+ * Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+package net.sf.mzmine.util.R;
+
+/**
+ * @description TODO
+ * @author Gauthier Boaglio
+ * @date Dec 07, 2014
+ */
```

Comment:
Please remove these tags such as @author and @date, because these are confusing.
GitHub provides much better data on authorship and dates of each change.